client/src/modules/stock/lots/registry.js

```diff
@@ -235,6 +207,10 @@ function StockLotsController(
     vm.loading = !vm.loading;
   }
 
+  function orderByDepot(rowA, rowB) {
+    return rowA.depot_text > rowB.depot_text ? 1 : -1;
```

Comment:
This is a non-blocking comment, in terms of performances, if the use of $timeout works what can be acceptable in the of performances : sort data or use of $timeout ?

Reply:
Timeout didn't work for me.  Did it work for you?

Reply:
I am indifferent to whether we use sorting or $timeout in the short run.  In the long run, I think we probably need to sort on the server in SQL first, then we'll figure out what is going on with the grouping issues :/